portfoliostyle.css

```diff
@@ -0,0 +1,438 @@
+
+
+* {
+	box-sizing: border-box;
+	font-family: 'Nunito', sans-serif;
+}
+
+section, footer div {
+	width: 65%;
+	margin: 0 auto;
+}
+
+body {
+	background-color: #D3D3D3;
+	background-image: linear-gradient(#808080, #FFFFFF);
+	background-position: top center;
+}
+
+/*most element styles*/
+
```

Comment:
Having the nav background change colour as well as the links overloads the brain a bit

Reply:
Is this personal preference or in relation to flashing?

Reply:
Amended to not change with hover